transformer_engine/pytorch/utils.py

```diff
@@ -329,6 +330,16 @@ def round_up_to_nearest_multiple(value, multiple):
     return ((value + multiple - 1) // multiple) * multiple
 
 
+def needs_quantized_gemm(obj, rowwise=True):
+    """Used to check if obj will need quantized gemm or normal gemm."""
+    if isinstance(obj, DebugQuantizedTensor):
+        return (
+            type(obj.get_tensor(not rowwise))  # pylint: disable=unidiomatic-typecheck
+            is not torch.Tensor
+        )
+    return type(obj) is not torch.Tensor  # pylint: disable=unidiomatic-typecheck
```

Comment:
```suggestion
        return not isinstance(obj.get_tensor(not rowwise), torch.Tensor)
    return not isinstance(obj, torch.Tensor)
```

Reply:
@ksivaman I don't believe this suggestion is correct. Basically you want to differentiate here between regular torch.Tensor and quantized tensor (which could be a subclass of torch.Tensor). In my internal quantizer PR #1638 I introduce a function to check if it is quantized tensor, maybe that would be better instead.